run

```diff
@@ -270,6 +292,28 @@ def build_swift_toolchain(workspace, args):
         raise common.UnsupportedPlatform
     common.check_execute(build_command, timeout=9999999)
 
+def build_clang(workspace, source_path):
+    build_path = os.path.join(workspace, 'build_clang_source_compat')
+    source_path = os.path.join(source_path, 'llvm')
+    common.check_execute(['mkdir', '-p', build_path])
+
+    with common.DirectoryContext(build_path):
+        try:
+            ninja_path = common.check_execute_output(['xcrun', '--find', 'ninja']).strip().decode("utf-8")
+            cmake_command = [
+                'xcrun', 'cmake', '-G', 'Ninja',
+                '-DCMAKE_MAKE_PROGRAM={}'.format(ninja_path),
+                '-DLLVM_ENABLE_PROJECTS=clang;llvm',
+                '-DCMAKE_BUILD_TYPE=Release',
```

Comment:
Swift decides whether to build a `Debug`, `Debug+Asserts`, `Release` or `Release+Asserts` version of itself depending on whether `--debug` and `--assertions` are specified.
 https://github.com/apple/swift-source-compat-suite/blob/43abe0b6035e28088e9bd049869f359467e1d2d4/run#L226-L233
Would it make sense to do the same here? 
If yes, the argument descriptions of the `--debug` and `--assertions` flags need to be updated.
If not, I think it would make sense to add a note to the `clang-source-path`, telling that `clang` will always be build in `Release` mode without assertions.

Reply:
I updated so that the build type is determined by the debug and assertions flags. I've also updated `--debug` and `--assertions` description that it may also apply to clang.